.github/workflows/publish.yml

```diff
@@ -33,14 +33,6 @@ jobs:
         uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
         with:
           path: dist/*
-      - name: upload package to GitHub release
-        uses: svenstaro/upload-release-action@2728235f7dc9ff598bd86ce3c274b74f802d2208 # v2
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN}}
-          file: dist/*
-          tag: ${{ github.ref }}
-          overwrite: true
-          file_glob: true
```

Comment:
this was a new feature, i can't get it to work, and i don't think its necessary. it would upload the python whl file to the GH release page, but users should be getting this from PyPI anyways, so not very important.